README.md

```diff
@@ -1,244 +1,36 @@
-<!--===- README.md 
-  
-   Part of the LLVM Project, under the Apache License v2.0 with LLVM Exceptions.
-   See https://llvm.org/LICENSE.txt for license information.
-   SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
-  
--->
+# 301 - Moved
 
-# F18
+F18 is now part of LLVM.
```

Comment:
Should you also say 
-> that is available in the flang directory
-> renamed and available as flang in LLVM.

Reply:
I believe I should only say something like this:

Reply:
This wording works for me

Reply:
Done!